### ISSUE TITLE
couldnt run the example folder for testing and exploration

### ISSUE THREAD

**Issue by sath26** [joanpablo/reactive_forms#124]
```
/C:/tools/flutter/.pub-cache/hosted/pub.dartlang.org/reactive_forms_widgets-0.3.9/lib/reactive_dropdown_search/reactive_dropdown_search.dart:33:41: Error: Expected 2 type arguments.
class ReactiveDropdownSearch<T> extends ReactiveFormField<T> {
                                        ^
/C:/tools/flutter/.pub-cache/hosted/pub.dartlang.org/reactive_forms_widgets-0.3.9/lib/reactive_touch_spin/reactive_touch_spin.dart:17:36: Error: Expected 2 type arguments.
class ReactiveTouchSpin<T> extends ReactiveFormField<T> {
                                   ^
/C:/tools/flutter/.pub-cache/hosted/pub.dartlang.org/reactive_forms_widgets-0.3.9/lib/reactive_segmented_control/reactive_segmented_control.dart:16:43: Error: Expected 2 type arguments.
class ReactiveSegmentedControl<T> extends ReactiveFormField<T> {
                                          ^
/C:/tools/flutter/.pub-cache/hosted/pub.dartlang.org/reactive_forms_widgets-0.3.9/lib/reactive_date_time_picker/reactive_date_time_picker.dart:37:38: Error: Expected 2 type arguments.
class ReactiveDateTimePicker extends ReactiveFormField<DateTime> {
                                     ^
/C:/tools/flutter/.pub-cache/hosted/pub.dartlang.org/reactive_forms_widgets-0.3.9/lib/reactive_date_range_picker/reactive_date_range_picker.dart:32:39: Error: Expected 2 type arguments.
class ReactiveDateRangePicker extends ReactiveFormField<DateTimeRange> {
                                      ^
/C:/tools/flutter/.pub-cache/hosted/pub.dartlang.org/reactive_forms_widgets-0.3.9/lib/reactive_image_picker/reactive_image_picker.dart:42:35: Error: Expected 2 type arguments.
class ReactiveImagePicker extends ReactiveFormField<ImageFile> {
                                  ^
lib/samples/reactive_form_widgets_sample.dart:26:15: Error: A value of type 'ReactiveDropdownSearch<String>' can't be assigned to a variable of type 'Widget'.
 - 'ReactiveDropdownSearch' is from 'package:reactive_forms_widgets/reactive_dropdown_search/reactive_dropdown_search.dart' ('/C:/tools/flutter/.pub-cache/hosted/pub.dartlang.org/reactive_forms_widgets-0.3.9/lib/reactive_dropdown_search/reactive_dropdown_search.dart').
 - 'Widget' is from 'package:flutter/src/widgets/framework.dart' ('/C:/tools/flutter/packages/flutter/lib/src/widgets/framework.dart').

              ReactiveDropdownSearch(
              ^
lib/samples/reactive_form_widgets_sample.dart:42:15: Error: A value of type 'ReactiveDropdownSearch<String>' can't be assigned to a variable of type 'Widget'.
 - 'ReactiveDropdownSearch' is from 'package:reactive_forms_widgets/reactive_dropdown_search/reactive_dropdown_search.dart' ('/C:/tools/flutter/.pub-cache/hosted/pub.dartlang.org/reactive_forms_widgets-0.3.9/lib/reactive_dropdown_search/reactive_dropdown_search.dart').
 - 'Widget' is from 'package:flutter/src/widgets/framework.dart' ('/C:/tools/flutter/packages/flutter/lib/src/widgets/framework.dart').
              ReactiveDropdownSearch<String>(
              ^
lib/samples/reactive_form_widgets_sample.dart:82:15: Error: A value of type 'ReactiveTouchSpin<double>' can't be assigned to a variable of type 'Widget'.
 - 'ReactiveTouchSpin' is from 'package:reactive_forms_widgets/reactive_touch_spin/reactive_touch_spin.dart' ('/C:/tools/flutter/.pub-cache/hosted/pub.dartlang.org/reactive_forms_widgets-0.3.9/lib/reactive_touch_spin/reactive_touch_spin.dart').
 - 'Widget' is from 'package:flutter/src/widgets/framework.dart' ('/C:/tools/flutter/packages/flutter/lib/src/widgets/framework.dart').
              ReactiveTouchSpin<double>(

              ^
lib/samples/reactive_form_widgets_sample.dart:96:15: Error: A value of type 'ReactiveTouchSpin<double>' can't be assigned to a variable of type 'Widget'.
 - 'ReactiveTouchSpin' is from 'package:reactive_forms_widgets/reactive_touch_spin/reactive_touch_spin.dart' ('/C:/tools/flutter/.pub-cache/hosted/pub.dartlang.org/reactive_forms_widgets-0.3.9/lib/reactive_touch_spin/reactive_touch_spin.dart').
 - 'Widget' is from 'package:flutter/src/widgets/framework.dart' ('/C:/tools/flutter/packages/flutter/lib/src/widgets/framework.dart').

```

**Comment by joanpablo** [comment]
Hi @sath26 @vasilich6107,

Yes you are right the examples folder depends on a library that is not using the latest version of Reactive Forms, in the next few days we will update the example folder with latest dependencies.

**Comment by vasilich6107** [comment]
Hi @sath26 
We are moving to the monorepo with separate packages for every field
You can find all links in readme https://github.com/artflutter/reactive_forms_widgets
The `reactive_forms_widgets` is deprecated.

Next week I'm going to release all widgets with support of null safety

**Comment by sath26** [comment]
@vasilich6107 looking forward for the release asap! thank u!

**Comment by vasilich6107** [comment]
@sath26 all packages migrated to null safety except `reactive_dropdown_search`
PR pending https://github.com/salim-lachdhaf/searchable_dropdown/pull/139

**Comment by joanpablo** [comment]
Hi @sath26 and @vasilich6107 

I have released a new version 10.1.0. In this version, I have temporarily disabled the dependency to the package *reactive_forms_widgets* until this package is fully migrated to null-safety. Now you can run the example project without troubles.

Thanks best regards
